app/(auth)/logout/Logout.tsx

```diff
@@ -41,8 +41,10 @@ export default function Logout(props: Props) {
     const logout = async () => {
       await logoutHandler();
     };
-    logout();
-  }, [props.fakeSessionToken]);
+    logout().catch((error) => {
+      setOnError(error.message);
+    });
+  });
```

Comment:
is an empty dependency array (`[]`) actually the correct second parameter here? `useEffect` without second parameter runs on every render, almost never what users want

Reply:
https://github.com/upleveled/graphql-example-spring-2023-austria-vienna/commit/03a29898f67ab8a9759782d3a71c227956b16952